tidb-cloud/import-csv-files.md

```diff
@@ -74,77 +74,77 @@ Because CSV files do not contain schema information, before importing data from
 
-1. Navigate to the **Clusters** page.
-2. Locate your target cluster, and click **Import Data** in the upper-right corner of the cluster area. The **Data Import Task** page is displayed.
+1. Log in to the [TiDB Cloud](https://tidbcloud.com/signin) console, and navigate to the **Clusters** page.
 
-    > **Tip:**
-    >
-    > Alternatively, you can also click the name of your target cluster on the **Clusters** page and click **Import Data** in the upper-right corner.
+2. Locate your target cluster, click **...** in the upper-right corner of the cluster area, and select **Import Data**. The **Data Import** page is displayed.
 
-3. On the **Data Import Task** page, provide the following information.
+3. On the **Data Import** page, provide the following information.
 
-    - **Data Source Type**: select the type of the data source.
-    - **Bucket URL**: select the bucket URL where your CSV files are located.
     - **Data Format**: select **CSV**.
-    - **Setup Credentials** (This field is visible only for AWS S3): enter the Role ARN value for **Role-ARN**.
-    - **CSV Configuration**: check and update the CSV specific configurations, including separator, delimiter, header, not-null, null, backslash-escape, and trim-last-separator. You can find the explanation of each CSV configuration right beside these fields.
-
-        > **Note:**
-        >
-        > For the configurations of separator, delimiter, and null, you can use both alphanumeric characters and certain special characters. The supported special characters include `\t`, `\b`, `\n`, `\r`, `\f`, and `\u0001`.
-
-    - **Target Cluster**: fill in the **Username** and **Password** fields.
-    - **DB/Tables Filter**: if you want to filter which tables to be imported, you can specify one or more table filters in this field, separated by `,`.
-
-        For example:
+    - **Location**: select the location where your CSV files are located.
+    - **Bucket URL**: select the bucket URL where your CSV files are located.
+    - **Role-ARN**: (This field is visible only for AWS S3): enter the Role ARN value for **Role-ARN**.
+    - **Target Cluster**: shows the cluster name and the region name.
 
-        - `db01.*`: all tables in the `db01` database will be imported.
-        - `db01.table01*,db01.table02*`: all tables starting with `table01` and `table02` in the `db01` database will be imported.
-        - `!db02.*`: except the tables in the `db02` database, all other tables will be imported. `!` is used to exclude tables that do not need to be imported.
-        - `*.*` : all tables will be imported.
+    If the location of the bucket is different from your cluster, confirm the compliance of cross region. Click **Next**. 
+    
+    TiDB Cloud starts validating whether it can access your data in the specified bucket URL. If you get the `AccessDenied` error, see [Troubleshoot Access Denied Errors during Data Import from S3](/tidb-cloud/troubleshoot-import-access-denied-error.md).
```

Comment:
```suggestion
    TiDB Cloud starts validating whether it can access your data in the specified bucket URL. After validation, TiDB Cloud tries to scan all the files in the data source using the default file naming pattern, and returns a scan summary result on the left side of the next page. If you get the `AccessDenied` error, see [Troubleshoot Access Denied Errors during Data Import from S3](/tidb-cloud/troubleshoot-import-access-denied-error.md).
```